Utils/EventsNames.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+#define STOP_STRESSER	L"Global\\StopStresser"
```

Comment:
Add class named `StresserConfiguration` (like `ServerConfiguration`) and move this constants to it.
The `Utils` is not specific related to `Stresser` project.

Reply:
First step: Move to Stresser project 
Second step: Create configuration class